packages/framework/src/Model/Complaint/Status/ComplaintStatusFacade.php

```diff
@@ -33,6 +36,10 @@ public function create(ComplaintStatusData $complaintStatusData): ComplaintStatu
         $this->em->persist($complaintStatus);
         $this->em->flush();
 
+        $this->mailTemplateFacade->createMailTemplateForAllDomains(
```

Comment:
the complaint status is not associated with the mail template this way Bw, the same problem is with the order statuses. I suggest fixing both usages in a separate commit by modifying the method like this:
```
public function createMailTemplateForAllDomains(string $name, ?OrderStatus $orderStatus = null, ?ComplaintStatus $complaintStatus = null): void
```

do not forget the upgrade notes then, please :pray:

Reply:
I have changed it, is it ok this way?

Reply:
yes, awesome!